ephios/settings.py

```diff
@@ -189,22 +191,25 @@
 ADMINS = getaddresses([env("ADMINS")])
 
 # logging
-LOGGING = {
-    "version": 1,
-    "disable_existing_loggers": True,
-    "handlers": {"mail_admins": {"level": "ERROR", "class": "django.utils.log.AdminEmailHandler"}},
-    "loggers": {
-        "django": {
+if not DEBUG:
```

Comment:
So I was annoyed that I didn't get a stacktrace, but a console email for errors.